docs/preprocess-templates/synchrotron_beta_logpoltens.ipynb

```diff
@@ -0,0 +1,325 @@
+   "execution_count": null,
+   "id": "a1f4fb6a-03a5-46dc-81bf-cd45194f736e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_dir = Path(\"production-data/synch\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "abea969d-d50a-45d3-ba66-3d4fc6fe0d21",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_nside = 8192\n",
+    "output_lmax = 2 * output_nside\n",
+    "output_ell = np.arange(output_lmax + 1)\n",
+    "output_cl_norm = output_ell * (output_ell + 1) / np.pi / 2\n",
+    "output_cl_norm[0] = 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5ab2cbe-8b48-4f06-9ee1-309750b35663",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "name=\"temperature\"\n",
+    "modulate_amp = hp.alm2map(hp.read_alm(output_dir / \"raw\" / f\"synch_{name}_modulation_alms_lmax{int(lmax):d}.fits\").astype(complex), output_nside)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8739464f-3c93-48dd-a59e-15f7f85a0c76",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "template_dir = \"/global/project/projectdirs/cmb/www/pysm-data/pysm_2/\"\n",
+    "beta_s_pysm = hp.read_map(template_dir + \"synch_beta.fits\")\n",
+    "print(beta_s_pysm.mean())\n",
+    "beta_s_pysm_nomono = hp.remove_monopole(beta_s_pysm)\n",
+    "print(beta_s_pysm_nomono.mean())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56bae391-7618-44cc-945e-fbf27023e9fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ell = np.arange(lmax + 1)\n",
+    "cl_spass_fit = 10 ** (-0.82655497) * ell ** (-2.60860208)\n",
+    "cl_spass_fit[0] = 0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "736969cf-d2c3-4361-972f-41b5d87a99e7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pysm3.utils import sigmoid"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4e0b6047-4011-4fe5-b1f3-08f10951266b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ell_fit_low = {\"TT\": 10, \"EE\": 10, \"BB\": 10}\n",
+    "ell_fit_high = {\"TT\": 36, \"EE\": 36, \"BB\": 36}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d805b7cb-c40b-4fb3-83bf-a055bc6d4a35",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "np.random.seed(444)\n",
+    "\n",
+    "bs_ss_alm = hp.synalm(cl_spass_fit, lmax=output_lmax)\n",
+    "alm_bs = hp.map2alm(beta_s_pysm_nomono, lmax=output_lmax, use_pixel_weights=True)\n",
+    "\n",
+    "bs_LS_alm = np.empty_like(alm_bs)\n",
+    "\n",
+    "sig_func = sigmoid(ell, x0=ell_fit_high[\"TT\"] - 15, width=ell_fit_high[\"TT\"] / 10)\n",
+    "bs_LS_alm = hp.almxfl(alm_bs, np.sqrt(1.0 - sig_func))\n",
+    "bs_ss_alm = hp.almxfl(bs_ss_alm, np.sqrt(sig_func))\n",
+    "bs_ss = hp.alm2map(bs_ss_alm, nside=output_nside)\n",
+    "\n",
+    "bs_ss *= modulate_amp\n",
```

Comment:
@giuspugl you are multiplying the small scales of beta by the modulation amplitude twice.
here